src/schemas/backend/UpdateTimesheet.ts

```diff
@@ -0,0 +1,73 @@
+import { z } from "zod";
+import { RowSchema, CommentSchema, ScheduledRowSchema } from "../RowSchema"
+import * as dbtypes from './Timesheet'
+
+/*
+-------------------------------------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------------------------------------
+---------------------------------DELETE WHEN MONOREPO CREATED------------------------------------------------------
+-------------------------------------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------------------------------------
+
+*/
+
+// Currently supported timesheet operations 
+export const enum TimesheetOperations {
+    INSERT = "INSERT", 
+    UPDATE = "UPDATE", 
+    DELETE = "DELETE", 
+    STATUS_CHANGE = "STATUS_CHANGE", 
+    CREATE_TIMESHEET = "CREATE_TIMESHEET"
+} 
+
+
+
+export const enum TimesheetListItems {
+    TABLEDATA = "TABLEDATA", 
+    SCHEDULEDATA = "SCHEDULEDATA", 
+    WEEKNOTES = "WEEKNOTES"
+}
+
+const availableListTypes = z.enum([TimesheetListItems.TABLEDATA, TimesheetListItems.SCHEDULEDATA, TimesheetListItems.WEEKNOTES])
+
+export const DeleteRequest = z.object({
+    Type: availableListTypes, 
+    Id: z.string() 
```

Comment:
Could you leave a tiny tiny comment explaining what the 'id' is here? I know that it's pretty much the same as what you explain for the `UpdateRequest`, but might be helpful to explain it for all request types in case people get confused.

Reply:
updated on backend portion of repo.